src/Symfony/Component/PropertyAccess/Annotation/Property.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Annotation;
+
+/**
+ * Property accessor configuration annotation.
+ *
+ * @Annotation
+ * @Target({"PROPERTY"})
+ *
+ * @author Luis Ramón López <lrlopez@gmail.com>
+ */
+class Property
```

Comment:
@dunglas @webmozart :+1: for having only one annotation. ``@Property`` looks weird on a method, maybe ``@Accessor`` ?